File: easy/cheering/cheering.rb
"""
@author	André Peralta
@version 20/05/2021
"""

def cheering yards

	if yards < 1
		print "shh"
	elsif yards >= 1 and yards < 10
		print "Ra!"*yards 
	elsif yards > 10
		print 'High Five'
	end
end

puts "Number of yards: "
yards = gets.to_i

cheering(yards)

File: easy/cheering/cheering.c
#include <stdio.h>
#include <math.h>

void main() {

    int yards;

    scanf("%d",&yards);

    if(yards > 10) printf("High Five");

    else if(yards < 1 )printf("shh");
    else{
        while(yards >0){
            printf("Ra!");
            yards--;
        }
    }
}
